Dijkstra Algorithm/README.md

```diff
@@ -0,0 +1,287 @@
+# Dijkstra's algorithm
+
+This [algorithm](https://en.wikipedia.org/wiki/Dijkstra%27s_algorithm) was invented in 1956 by Edsger W. Dijkstra. 
+
+It can be used, when you have one source vertex and want to find the shortest paths to all other vertices in the graph.
+
+The best example is road network. If you wnat to find the shortest path from your house to your job, then it is time for the Dijkstra's algorithm.
+
+I have created **VisualizedDijkstra.playground** to improve your understanding of the algorithm's flow. Besides, below is step by step algorithm's description.
+
+Let's imagine, you want to go to the shop. Your house is A vertex and there are 4 possible stores around your house. How to find the closest one/ones? Luckily, you have graph, that connects your house with all these stores. So, you know what to do :)
+
+## Initialization
+
+When the algorithm starts to work initial graph looks like this:
+
+<img src="Images/image1.png" height="250" />
+
+The table below represents graph state:
```

Comment:
Instead of jumping straight to a table, gradually warm up the reader by introducing the concept of a weighted graph. This is where you update your diagram with more information:

<img width="682" alt="screen shot 2017-05-01 at 5 17 56 pm" src="https://cloud.githubusercontent.com/assets/11384608/25599581/24b6446e-2e92-11e7-9481-768dadcdfcb2.png">

For simplicity, tell the reader this is only a one way street (directed graph).